site/docs/v1/tech/messengers/index.adoc

```diff
@@ -0,0 +1,33 @@
+---
+layout: doc
+title: Messengers
+description: Adding and Configuring Messengers
+---
+
+[NOTE.since]
+====
+Available since 1.26.0
+====
+
+== Messengers
+
+Messengers are used to send messages from within FusionAuth to other systems. They can be customized using link:/docs/v1/tech/themes/[themes].
+
+== Set Up a Messenger
+
+In FusionAuth, you can set up three types of messengers:
+
+- link:twilio-messenger[Set Up a Twilio Messenger]
+- link:kafka-messenger[Set Up a Kafka Messenger]
+- link:generic-messenger[Set Up a Generic Messenger]
+
+To configure a messenger, navigate to [breadcrumb]#Settings-> Messengers# and click on the appropriate messenger type in the top right.
+
+== Create a Messenger Template
+
+Message templates are a powerful way to customize messengers and the communication contained within. FusionAuth uses FreeMarker for all templating.
```

Comment:
Link to here: https://fusionauth.io/docs/v1/tech/email-templates/message-templates/